algebra-kernels/src/polycommit.rs

```diff
@@ -0,0 +1,183 @@
+use algebra_cl_gen::gpu::{GPUError, GPUResult, kernel_polycommit, get_prefix_map};
```

Comment:
Maybe it's worth to have a couple of unit tests asserting equality between the GPU result and the CPU one ?

Reply:
https://github.com/HorizenLabs/poly-commit/blob/dev/src/ipa_pc/mod.rs#L1175